lib/jsdom/living/custom-elements/CustomElementRegistry-impl.js

```diff
@@ -51,7 +52,7 @@ function isConstructor(value) {
   }
 }
 
-// https://html.spec.whatwg.org/#customelementregistry
+// https://html.spec.whatwg.org/multipage/custom-elements.html#customelementregistry
```

Comment:
Please don't make unrelated changes in the PR. You can file issues asking if they are OK but it is not good to do so unsolicited.